scripts/kube-init.sh

```diff
@@ -32,7 +32,8 @@ setenforce 0
 
 # Mount root to fix dns issues
 # Define $HOME since somehow this is not defined
-HOME=/home/travis
+# Changed from travis to GH Actions agent default user
+HOME=/home/runner 
```

Comment:
Isn't `$HOME` already defined in GHA?  https://docs.github.com/en/actions/learn-github-actions/environment-variables#naming-conventions-for-environment-variables

Reply:
```
- run: echo "$HOME"
/home/runner

```

Reply:
Yes, thank you! Whilst testing I thought $HOME is required for TOX run but I have tested by removing $HOME and CI runs fine. Updated!